beta/project-griffon/set-up-project-griffon.md

```diff
@@ -118,13 +122,19 @@ func application(_ application: UIApplication, didFinishLaunchingWithOptions lau
 
 ### Start a Project Griffon session
 
-After the extension has been registered, start a Project Griffon session by using the following API:
+1. [Create](https://aep-sdks.gitbook.io/docs/beta/project-griffon/using-project-griffon#1-create-a-session) a new session in the [griffon UI](https://griffon.adobe.com/) by providing a base url for your application.
```

Comment:
i'm not a fan of using links like this; it makes the step text confusing, and I see it all throughout the AEP docs...sigh. 

Please reword as follows:
"1. Create a session in the [Griffon UI](https://griffon.adobe.com/) by typing a base URL  for your application. (I removed "new" because you cannot create anything but a new session!)

  For more information about creating a session, see [Create a session](https://aep-sdks.gitbook.io/docs/beta/project-griffon/using-project-griffon#1-create-a-session).